apps/judicial-system/web/src/components/InfoCard/InfoCard.strings.ts

```diff
@@ -25,6 +25,12 @@ export const infoCardCaseScheduled = defineMessages({
     defaultMessage: 'Á dagskrá',
     description: 'Notaður sem texti sem tilgreinir að málið sé á dagskrá',
   },
+  postponed: {
```

Comment:
Consider changing the namespace of these strings and moving to core

Reply:
Good catch, I have made the change.